c++/src/Statistics.hh

```diff
@@ -23,10 +23,17 @@
 #include "orc/OrcFile.hh"
 #include "orc/Reader.hh"
 
+#include "Timezone.hh"
 #include "TypeImpl.hh"
 
 namespace orc {
 
+  struct StatContext {
+    bool correctStats;
+    const Timezone* writerTimezone;
+    StatContext() : correctStats(false), writerTimezone(NULL) {}
```

Comment:
Shouldn't the constructor take the correctStats and writerTimezone and make them const?